department-of-veterans-affairs/va.gov-cms
# VACMS-15142: Page-specific Q&A Report

## Description

Relates to #15142 

This is a content report at /admin/content/qa to show all Page-specific Q&A content that uses that template.
Note: There is Q&A-related text in other places (using other types of main content blocks like rich text and accordions) that did not use the paragraph template for page-specific Q&As, such as on the Resources and Support content types. This PR and report does not cover those instances.

## Testing done
Local functional

## QA steps
Log in as a user with the access that CAIA content team has.
Navigate to [/admin/content/qa](https://cms-80d0prfrxp0pohhtro3rzzrlo2bjdxih.ci.cms.va.gov/admin/content/qa).
Content is populated for the following node types, with approximately 1836 total, give or take, will be a little different because of varying test data.

- [ ] 737 Benefit Detail Questions
- [ ] 31 CMS KB Article Questions
- [ ] 85 CLP Questions
- [ ] 12 Centralized Content Questions
- [ ] 970 VAMC Detail Questions

Ensure that you are able to:
- [ ] click on a node to view the listed content
- [ ] there are columns for section, node title and link, and node type

### Definition of Done


### Select Team for PR review



### Is this PR blocked by another PR?


### Does this PR need review from a Product Owner


### CMS user-facing announcement

Is an announcement needed to let editors know of this change?
